closed/GensrcJ9JCL.gmk

```diff
@@ -36,7 +36,7 @@ ifeq (true,$(OPENJ9_ENABLE_DDR))
   JppSourceDirs += $(OPENJ9_TOPDIR)/debugtools/DDR_VM/src
 endif # OPENJ9_ENABLE_DDR
 
-JPP_TAGS := PLATFORM-$(OPENJ9_PLATFORM_CODE) OPENJDK_THREAD_SUPPORT
```

Comment:
Are you going to remove this flag from the OpenJ9 jpp_configuration.xml after the extensions changes are merged?

Reply:
My bad, I see https://github.com/eclipse-openj9/openj9/pull/16521 now.